flang/lib/Lower/Bridge.cpp

```diff
@@ -1480,18 +1480,22 @@ class FirConverter : public Fortran::lower::AbstractConverter {
       return;
     if (const auto *details =
             sym.detailsIf<Fortran::semantics::ObjectEntityDetails>()) {
-      if (details->init())
+      if (details->init()) {
         global = builder->createGlobal(
             toLocation(), genType(sym), globalName, false,
             [&](Fortran::lower::FirOpBuilder &builder) {
               auto initVal = genExprValue(details->init().value());
               builder.create<fir::HasValueOp>(toLocation(), initVal);
             });
-      else
+      }
+      else {
```

Comment:
The formatting here looks off. Remember to use clang-format. (Also, the braces are not needed here. See the LLVM style guide.)